service/datasets/evaluation/PUMS_1000.csv

```diff
@@ -0,0 +1,1001 @@
+age,sex,educ,race,income,married

```

Comment:
We should not upload the dataset. Can we download it in tests/conftest.py and add a gitignore for *.csv,

Reply:
Downloading via conftest.py. Updated .gitignore with the entry for the PUMS_1000.csv like the other CSV files already mentioned in it.